packages/mail/lib/mailLog.js

```diff
@@ -54,32 +58,40 @@ const send = async ({
   if (!sendFunc || !message || !email || !context || !context.pgdb) {
     throw new Error('missing input', { sendFunc, message, email, context })
   }
-  const onceFor = log && log.onceFor
+
   if (onceFor) {
     if (await wasSent(onceFor, context)) {
       debug('not sending (was sent already): %o', log)
       return
     }
   }
 
-  const info = {
-    ...log && log.info,
-    template,
-    message
+  const info = {}
+
+  if (log) {
+    Object.assign(info, log.info)
+  }
+
+  if (template) {
+    info.template = template
+  }
+
+  if (logMessage) {
+    info.message = message
   }
```

Comment:
Why did you change this code block? What's the advantage of this notation to the old one?

Reply:
Easier to test, read and stitch together.